source/_patterns/02-organisms/60-monthly-good/01-monthly-good-inputs.mustache

```diff
@@ -180,11 +180,11 @@
 			   data-abide-validator="sum"
 			   data-max-sum="10000"
 			   data-sum-inputs="deposit-amount-input,donation-amount-input">
-		<small class="error">
+		<small class="error zig">
 			<svg class="icon icon-error">
 				<use xlink:href="#icon-error"/>
 			</svg>
-			The maximum auto deposit allowed is $10,000. Please try again by entering in a smaller amount.
+			The maximum Monthly Good total is $10,000. Please try again by entering in a smaller amount.
 		</small>
```

Comment:
This is probably a question for @brandonatkiva, but the error text here doesn't fully explain to the user that the combination of their deposit_amount & donation_amount is what equals their "Monthly Good total" and not simply the deposit_amount entered.